src/js/wrap.h

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/logger.h"
+#include "historical_queries_interface.h"
+#include "kv/kv_types.h"
+#include "kv/tx.h"
+
+#include <memory>
+#include <quickjs/quickjs-exports.h>
+#include <quickjs/quickjs.h>
+
+namespace js
+{
+  extern JSClassID kv_class_id;
+  extern JSClassID kv_map_handle_class_id;
+  extern JSClassID body_class_id;
+
+  extern JSClassDef kv_class_def;
+  extern JSClassExoticMethods kv_exotic_methods;
+  extern JSClassDef kv_map_handle_class_def;
+  extern JSClassDef body_class_def;
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wc99-extensions"
+
+  extern void register_class_ids();
```

Comment:
I haven't seen `extern` used like that for function declarations before. Probably can be omitted?

Reply:
Yes, extern is the default, bad search and replace, fixed.